uvls/src/smt.rs

```diff
@@ -25,8 +25,8 @@ use tokio::{
 
 use tokio_util::sync::CancellationToken;
 use tower_lsp::lsp_types::*;
-pub mod smt_lib;
 mod parse;
+pub mod smt_lib;
```

Comment:
is this necessary? maybe leave it as it was to make the commit as clean as possible.